# Ontario Brain Institute Molecular Tools

Please select one of the dropdowns to browse tools used in molecular data collection, data curation and processing, and data analysis.

<details><summary>Data Collection Tools</summary>
&nbsp 
    
| Tool/Pipeline | Description | Requirements | Compute Location | Research Program(s) |
| ---------------- | ----------- | --------------------------- | ----------- | ---------|
| [LabKey](https://www.labkey.com/) | Software used for life science data management. Data is collected in data form. <details><summary>License</summary>Apache Version 2.0</details> <details><summary>Tool Citation(s)</summary>**General Use**: Nelson EK, Piehler B, Eckels J, Rauch A, Bellew M, Hussey P, Ramsay S, Nathe C, Lum K, Krouse K, Stearns D, Connolly B, Skillman T, Igra M. LabKey Server: An open source platform for scientific data integration, analysis and collaboration. BMC Bioinformatics 2011 Mar 9; 12(1): 71. <br></br> **Proteomics**: Rauch A, Bellew M, Eng J, Fitzgibbon M, Holzman T, Hussey P, Igra M, Maclean B, Lin CW, Detter A, Fang R, Faca V, Gafken P, Zhang H, Whitaker J, States D, Hanash S, Paulovich A, McIntosh MW: Computational Proteomics Analysis System (CPAS):  An Extensible, Open-Source Analytic System for Evaluating and Publishing Proteomic Data and High Throughput Biological Experiments. Journal of Proteome Research 2006, 5:112-121. <br></br> **Flow**: Shulman N, Bellew M, Snelling G, Carter D, Huang Y, Li H, Self SG, McElrath MJ, De Rosa SC: Development of an automated analysis system for data from flow cytometric intracellular cytokine staining assays from clinical vaccine trials. Cytometry 2008, 73A:847-856.</details> <details><summary>**Relevant Publications**</summary>Is it needed for LabKey?...</details>| N/A | [Brain-CODE](https://www.braincode.ca/) | CAN-BIND |
| [ClinVar](https://www.ncbi.nlm.nih.gov/clinvar/) | Scientifically supported public archive of reports of the relationships among human variations and phenotypes <details><summary>Tool Citation(s)</summary>Landrum MJ, Lee JM, Benson M, Brown GR, Chao C, Chitipiralla S, Gu B, Hart J, Hoffman D, Jang W, Karapetyan K, Katz K, Liu C, Maddipatla Z, Malheiro A, McDaniel K, Ovetsky M, Riley G, Zhou G, Holmes JB, Kattman BL, Maglott DR. ClinVar: improving access to variant interpretations and supporting evidence. Nucleic Acids Res . 2018 Jan 4. PubMed PMID: 29165669 </details> <details><summary>Relevant Publications</summary>[Landrum et al., 2018](https://doi.org/10.1093/nar/gkx1153), [Carvalho-Silva et al., 2019](https://doi.org/10.1093/nar/gky1133), [Landrum et al., 2020](https://doi.org/10.1093/nar/gkz972), [Piñero et al., 2020](https://doi.org/10.1093/nar/gkz1021)</details> | N/A | Web-based| EpLink |
| [OMIM](https://www.omim.org/)| A comprehensive manual of human genes and genetic phenotypes <details><summary>Tool Citation(s)</summary>Online Mendelian Inheritance in Man, OMIM®. McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University (Baltimore, MD), {date}. World Wide Web URL: https://omim.org/ <br></br> Details on how to cite specific articles from OMIM can be found [here](https://www.omim.org/help/faq#1_8:~:text=Citing%20a%20specific%20entry%20in%20OMIM%3A)</details> <details><summary>Relevant Publications</summary>[Hamosh et al., 2005](https://doi.org/10.1093/nar/gki033), [McKusick, 2007](https://doi.org/10.1086/514346), [Amberger et al., 2015](https://doi.org/10.1093/nar/gku1205), [Amberger et al., 2019](https://doi.org/10.1093/nar/gky1151)</details>| N/A |  Web-based | EpLink |
| [Varsome](https://varsome.com/) | A variant knowledge community, data aggregator, and variant data discovery tool <details><summary>**License**</summary></details><details><summary>Tool Citation(s)</summary>VarSome: the human genomic variant search engine, Christos Kopanos, Vasilis Tsiolkas, Alexandros Kouris, Charles E Chapple, Monica Albarca Aguilera, Richard Meyer, Andreas Massouras, Bioinformatics, Volume 35, Issue 11, 1 June 2019, Pages 1978–1980, https://doi.org/10.1093/bioinformatics/bty897 </details><details><summary>Relevant Publications</summary>[Kopanos et al., 2019](https://doi.org/10.1093/bioinformatics/bty897), [Kumar et al., 2019](https://doi.org/10.1016/j.parkreldis.2019.11.004), [Marcelino-Rodriguez et al., 2019](https://www.frontiersin.org/articles/10.3389/fgene.2019.00900), [Yap et al., 2020](https://doi.org/10.1158/2159-8290.CD-20-0163)</details>| N/A |  Web-based | EpLink |
 
</details>

<details><summary>Data Curation and Processing Tools</summary>
&nbsp
    
| Tool/Pipeline | Description | Requirements | Compute Location | Research Program(s) |
| ---------------- | ----------- | --------------------------- | ----------- | ---------|
| [Plink](https://github.com/GELOG/plink) | genome association analysis toolset that can be used to conduct a range of large scale analyses <details><summary>License</summary>GNU General Public License Version 2.0</details> <details><summary>Tool Citation(s)</summary>Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MAR, Bender D, Maller J, Sklar P, de Bakker PIW, Daly MJ & Sham PC (2007) PLINK: a toolset for whole-genome association and population-based linkage analysis. American Journal of Human Genetics, 81. </br><br>PLINK: Whole genome data analysis toolset. (n.d.). Retrieved August 4, 2022, from https://zzz.bwh.harvard.edu/plink/</details> <details><summary>Relevant Publications</summary>[Ge et al., 2009](https://doi.org/10.1038/nature08309), [Purcell et al., 2009](https://doi.org/10.1038/nature08185), [Yang et al., 2010](https://doi.org/10.1038/ng.608), [Yang et al., 2011](https://doi.org/10.1016/j.ajhg.2010.11.011)</details>| N/A | At the lab | CAN-BIND, ONDRI |
| [IMPUTE](https://mathgen.stats.ox.ac.uk/impute/impute_v2.html) | Computer program used for whole-genome imputation. <details><summary>License</summary> [Academic Use License](https://jmarchini.org/licence/)</details> <details><summary>Tool Citation(s)</summary>B. N. Howie, P. Donnelly, and J. Marchini (2009) A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. PLoS Genetics 5(6): e1000529 [Open Access Article] [Supplementary Material] </details> <details><summary>**Relevant Publications**</summary></details>| Currently only available for Mac and Linux operating systems. | At the lab | CAN-BIND |
| [Agilent 2200 Tapestation](https://www.agilent.com/en/product/automated-electrophoresis/tapestation-systems/tapestation-software/2200-tapestation-software-228269) | Software used for RNA Integrity checks. <details><summary>License</summary> Limited Use Label License</details> <details><summary>Tool Citation(s)</summary>2200 TapeStation Instrument (RRID:SCR_014994)</details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [Illumina HiSeq4000](https://www.illumina.com/systems/sequencing-platforms/hiseq-3000-4000.html) (discontinued)| Tool used for RNA sequencing. <details><summary>License</summary> Limited Use Label License</details> <details><summary>Tool Citation(s)</summary>Illumina HiSeq 3000/HiSeq 4000 System (RRID:SCR_016386)</details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [FASTXToolkit Trimmomatic](http://hannonlab.cshl.edu/fastx_toolkit/) | Tool used for quality and adapter trimming. <details><summary>License</summary> GNU General Public License Version 3.0 </details> <details><summary>Tool Citation(s)</summary>Bolger, A. M., Lohse, M., & Usadel, B. (2014). Trimmomatic: A flexible trimmer for Illumina Sequence Data. Bioinformatics, btu170.</details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [Tophat2 - bowtie2](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml) | Tool used to align the cleaned reads to reference genome (GRCh38). <details><summary>License</summary> GNU General Public License Version 3.0 </details> <details><summary>Tool Citation(s)</summary>Langmead B, Salzberg S. Fast gapped-read alignment with Bowtie 2. Nature Methods. 2012, 9:357-359.</details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [HTSeq-count (ref ENSEMBL)](https://htseq.readthedocs.io/en/release_0.11.1/index.html) | Tool used for quantification on each gene’s expression. <details><summary>License</summary> GNU General Public License Version 3.0 </details> <details><summary>Tool Citation(s)</summary>Simon Anders, Paul Theodor Pyl, Wolfgang Huber HTSeq — A Python framework to work with high-throughput sequencing data Bioinformatics (2014), in print, online at doi:10.1093/bioinformatics/btu638</details> <details><summary>**Relevant Publications**</summary></details>| Mac or Linux operating systems, Python 2.7 or >3.4, NumPy, Pysam, and Matplotlib | At the lab | CAN-BIND |
| [DESeq2 (R Package)](https://bioconductor.org/packages/release/bioc/html/DESeq2.html) | Tool used for the normalization of resulting gene matrix. <details><summary>License</summary> GNU General Public License Version 3.0 </details> <details><summary>Tool Citation(s)</summary>Love MI, Huber W, Anders S (2014). “Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.” Genome Biology, 15, 550. doi: 10.1186/s13059-014-0550-8.</details> <details><summary>**Relevant Publications**</summary></details> | N/A | At the lab | CAN-BIND |
| [ERCC's exceRpt small RNA-Seq pipeline](https://github.gersteinlab.org/exceRpt/) | Aligned the reads to the human genome and quantified the various miRNAs into counts. Can be run on [Genboree](http://genboree.org/site/) or [Docker](https://www.docker.com/). <details><summary>License</summary> GNU General Public License Version 3.0 </details> <details><summary>Tool Citation(s)</summary>Rozowsky, J., Kitchen, R. R., Park, J. J., Galeev, T. R., Diao, J., Warrell, J., Thistlethwaite, W., Subramanian, S. L., Milosavljevic, A., & Gerstein, M. (2019). exceRpt: A Comprehensive Analytic Platform for Extracellular RNA Profiling. Cell Systems, 8(4), 352-357.e3. https://doi.org/10.1016/j.cels.2019.03.004 </details> <details><summary>**Relevant Publications**</summary></details>| ~ 16 GB RAM, > 4 CPU cores| At the lab | CAN-BIND |
| [MEME Suite](https://meme-suite.org/meme/) | Tool used for consensus motif enrichment discovery analysis. <details><summary>License</summary> https://github.com/cinquin/MEME/blob/master/COPYING </details> <details><summary>Tool Citation(s)</summary> Timothy L. Bailey, James Johnson, Charles E. Grant, William S. Noble, "The MEME Suite", Nucleic Acids Research, 43(W1):W39-W49, 2015. [[full text](https://academic.oup.com/nar/article/43/W1/W39/2467905?login=false)] </br><br> Information on how to cite specific tools in MEME suite can be found [here](https://meme-suite.org/meme/doc/cite.html?man_type=web).</details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [Enrichr](https://maayanlab.cloud/Enrichr/) | Tool used for enrichment analysis. <details><summary>License</summary> GNU General Public License Version 3.0 </details> <details><summary>Tool Citation(s)</summary>Chen EY, Tan CM, Kou Y, Duan Q, Wang Z, Meirelles GV, Clark NR, Ma'ayan A. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. BMC Bioinformatics. 2013; 128(14). </br><br> Kuleshov MV, Jones MR, Rouillard AD, Fernandez NF, Duan Q, Wang Z, Koplev S, Jenkins SL, Jagodnik KM, Lachmann A, McDermott MG, Monteiro CD, Gundersen GW, Ma'ayan A. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. Nucleic Acids Research. 2016; gkw377 . </br><br> Xie Z, Bailey A, Kuleshov MV, Clarke DJB., Evangelista JE, Jenkins SL, Lachmann A, Wojciechowicz ML, Kropiwnicki E, Jagodnik KM, Jeon M, & Ma’ayan A. Gene set knowledge discovery with Enrichr. Current Protocols, 1, e90. 2021. doi: 10.1002/cpz1.90</details> <details><summary>**Relevant Publications**</summary></details> | N/A | At the lab | CAN-BIND |
| [ARCHS4](https://maayanlab.cloud/archs4/) | Tissue expression atlas. <details><summary>License</summary> Source Code: Apache License Version 2.0 </br><br>Gene expression files: Creative Commons Attribution 4.0 International License </details> <details><summary>Tool Citation(s)</summary>Lachmann A, Torre D, Keenan AB, Jagodnik KM, Lee HJ, Wang L, Silverstein MC, Ma’ayan A. Massive mining of publicly available RNA-seq data from human and mouse. Nature Communications 9. Article number: 1366 (2018), doi:10.1038/s41467-018-03751-6</details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [Infinium MethylationEPIC Beadchip](https://support.illumina.com/array/array_kits/infinium-methylationepic-beadchip-kit.html) | Tool used to assess genome-wide DNA methylation. | N/A | At the lab | CAN-BIND |
| [Chip Analysis Methylation Pipeline (ChAMP) Bioconductor (R package)](https://www.bioconductor.org/packages/devel/bioc/vignettes/ChAMP/inst/doc/ChAMP.html) | Tool used for pre-processing and analysis of raw microarray data. <details><summary>License</summary> GNU General Public License Version 3.0 </details> <details><summary>Tool Citation(s)</summary>Morris, T. J., Butcher, L. M., Feber, A., Teschendorff, A. E., Chakravarthy, A. R., Wojdacz, T. K., and Beck, S. (2014). Champ: 450k chip analysis methylation pipeline pg - 428-30. Bioinformatics, 30(3), 428-30. </br><br> More information on citing ChAMP can be found [here](https://www.bioconductor.org/packages/devel/bioc/vignettes/ChAMP/inst/doc/ChAMP.html#section-citing-champ:~:text=many%20other%20tools.-,7%20Citing%20ChAMP,-ChAMP%20is%20a). </details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [TopSpin 3.6](https://www.bruker.com/en/products-and-solutions/mr/nmr-software/topspin.html) | Tool used for 1H NMR Spectra processing. <details><summary>License</summary> Academic Use License </details> <details><summary>Tool Citation(s)</summary>TopSpin (RRID:SCR_014227)</details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| Meds Tool | Excel based tool that performs drug categorization.| N/A | At the lab | ONDRI |
| [Simoa](https://www.svarlifescience.com/knowledge/technologies/simoa-bead-technology#:~:text=Simoa%20is%20an%20ultra%2Dsensitive,in%20most%20common%20sample%20types.) | Tool used for molecular classification of amyloid and Tau proteins </br><br> can't seem to find license OR tool citation...| N/A | [Brain-CODE](https://www.braincode.ca/) | ONDRI |
| [Masshunter](https://www.agilent.com/en/product/software-informatics/mass-spectrometry-software?gclid=EAIaIQobChMI0drnxpWZ-QIV_T6tBh20DQ7pEAAYASAAEgJ7MfD_BwE&gclsrc=aw.ds) | Quantitation tool upstream of data upload <details><summary>License</summary>[Customary non-commercial license](https://www.agilent.com/cs/library/quickreference/public/Agilent_Software_EULA.pdf) </details> <details><summary>Tool Citation(s)</summary>Agilent Masshunter Quantitative Analysis software (RRID:SCR_015040)</details> <details><summary>Relevant Publications</summary>[Duncan et al., 2015](https://doi.org/10.1016/j.chembiol.2015.03.010), [Müller et al., 2019](https://doi.org/10.1038/s41592-019-0394-y), [Pu et al., 2020](https://doi.org/10.1111/nph.16552), [Boulias & Greer, 2021](https://doi.org/10.1007/978-1-0716-0876-0_7)</details>| N/A | At the lab | ONDRI |
| [Data Preparation (Shiny) App](https://github.com/ondri-nibs/dataprep_app) | Prepares ONDRI data for outlier analysis. <details><summary>License</summary>GNU General Public License Version 3.0</details> | R, RStudio, [GSVD](https://github.com/derekbeaton/GSVD) and [ours](https://github.com/derekbeaton/OuRS) R Packages | At the lab | ONDRI| 
| [Outliers (Shiny) App](https://github.com/ondri-nibs/outliers_app) | Performs outlier analyses on ONDRI data <details><summary>License</summary>GNU General Public License Version 3.2</details> | R, Rstudio | At the lab | ONDRI| 
| [Adobe Acrobat Pro 2020](https://www.adobe.com/ca/acrobat/complete-pdf-solution.html?mv=search&ef_id=EAIaIQobChMIz-2g35WZ-QIVIz2tBh1spgDtEAAYASAAEgLHzPD_BwE:G:s&s_kwcid=AL!3085!3!495116485612!e!!g!!adobe%20acrobat%20pro!12198071516!116786778653&gclid=EAIaIQobChMIz-2g35WZ-QIVIz2tBh1spgDtEAAYASAAEgLHzPD_BwE) | Software used to redact all personal health information (PHI) from genetic reports. <details><summary>License</summary>Named User License</details>| N/A | At the lab | EpLink |
</details>

<details><summary>Data Analysis Tools</summary>
&nbsp 
    
| Tool/Pipeline | Description | Requirements | Compute Location | Research Program(s) |
| ---------------- | ----------- | --------------------------- | ----------- | ---------|
| [Chip Analysis Methylation Pipeline (ChAMP) Bioconductor (R package)](https://www.bioconductor.org/packages/devel/bioc/vignettes/ChAMP/inst/doc/ChAMP.html) | Tool used for pre-processing and analysis of raw microarray data. <details><summary>License</summary> GNU General Public License Version 3.0 </details> <details><summary>Tool Citation(s)</summary>Morris, T. J., Butcher, L. M., Feber, A., Teschendorff, A. E., Chakravarthy, A. R., Wojdacz, T. K., and Beck, S. (2014). Champ: 450k chip analysis methylation pipeline pg - 428-30. Bioinformatics, 30(3), 428-30. </br><br> More information on citing ChAMP can be found [here](https://www.bioconductor.org/packages/devel/bioc/vignettes/ChAMP/inst/doc/ChAMP.html#section-citing-champ:~:text=many%20other%20tools.-,7%20Citing%20ChAMP,-ChAMP%20is%20a). </details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [miRwalk 2.0](http://mirwalk.umm.uni-heidelberg.de/) |Tool used for target prediction of miRNA cargo. <details><summary>**License**</summary> </details> <details><summary>Tool Citation(s)</summary>Sticht C, De La Torre C, Parveen A, Gretz N.: miRWalk: An online resource for prediction of microRNA binding sites. PLoS One. 2018 Oct 18;13(10): </details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [miRanda](https://cbio.mskcc.org/miRNA2003/miranda.html) |Tool used for target prediction of miRNA cargo. <details><summary>License</summary> GNU General Public License Version 2.0 </details> <details><summary>Tool Citation(s)</summary>A.J. Enright, B. John, U. Gaul, T. Tuschl, C. Sander, D.S. Marks; (2003) MicroRNA targets in Drosophila; Genome Biology 5(1):R1.</details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [RNA22 v2](https://cm.jefferson.edu/rna22/Interactive/) |Tool used for target prediction of miRNA cargo. <details><summary>**License**</summary></details> <details><summary>Tool Citation(s)</summary> Miranda KC, Huynh T, Tay Y, Ang YS, Tam WL, Thomson AM, Lim B, Rigoutsos I. (2006) A pattern-based method for the identification of MicroRNA binding sites and their corresponding heteroduplexes. Cell 126(6):1203-17. [[article](https://www.cell.com/cell/fulltext/S0092-8674%2806%2901099-3)] </details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [RNAHybrid](https://bibiserv.cebitec.uni-bielefeld.de/rnahybrid) |Tool used for target prediction of miRNA cargo. <details><summary>License</summary> GNU General Public License Version 2.0 </details> <details><summary>Tool Citation(s)</summary> Rehmsmeier, Marc and Steffen, Peter and Hoechsmann, Matthias and Giegerich, Robert Fast and effective prediction of microRNA/target duplexes RNA, RNA, 2004</details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [TargetScan](https://www.targetscan.org/vert_80/) |Tool used for target prediction of miRNA cargo. <details><summary>**License**</summary> </details> <details><summary>Tool Citation(s)</summary> Citation information can be found [here](https://www.targetscan.org/vert_80/#:~:text=To%20reference%20information%20from%20this%20database%2C%20please%20cite%20one%20of%20the%20following%20papers%3A). </details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [METAL](https://github.com/statgen/METAL) | Software used for metal-analysis and Cochran l test. <details><summary>**License**</summary>License information can be found [here](https://github.com/statgen/METAL/blob/master/LICENSE.twister).</details> <details><summary>Tool Citation(s)</summary> Willer, C. J., Li, Y., & Abecasis, G. R. (2010). METAL: Fast and efficient meta-analysis of genomewide association scans. Bioinformatics, 26(17), 2190–2191. https://doi.org/10.1093/bioinformatics/btq340  </details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [survival (R package)](https://rdrr.io/cran/survival/) | R package used to conduct Cox regression. <details><summary>License</summary>GNU General Public License Version 3.0</details> <details><summary>Tool Citation(s)</summary> Therneau T (2022). A Package for Survival Analysis in R. R package version 3.4-0, https://CRAN.R-project.org/package=survival. </br><br>Terry M. Therneau, Patricia M. Grambsch (2000). Modeling Survival Data: Extending the Cox Model. Springer, New York. ISBN 0-387-98784-3.  </details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [lme4 (R package)](https://rdrr.io/cran/lme4/) | R package used to conduct Linear mixed-effects models. <details><summary>License</summary>GNU General Public License Version 3.0</details> <details><summary>Tool Citation(s)</summary> Bates D, Mächler M, Bolker B, Walker S (2015). “Fitting Linear Mixed-Effects Models Using lme4.” Journal of Statistical Software, 67(1), 1–48. doi:10.18637/jss.v067.i01. </details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [FUMA v1.3.5e – MAGMA](https://fuma.ctglab.nl/) | Software used for genome-wide gene analysis. <details><summary> License </summary>Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International Public License</details> <details><summary>Tool Citation(s)</summary> K. Watanabe, E. Taskesen, A. van Bochoven and D. Posthuma. Functional mapping and annotation of genetic associations with FUMA. Nat. Commun. 8:1826. (2017). https://www.nature.com/articles/s41467-017-01261-5 </details> <details><summary>**Relevant Publications**</summary></details>| N/A | At the lab | CAN-BIND |
| [RStudio](https://www.rstudio.com/) |Software used to write scripts to conduct data analysis.| N/A | At the lab | ONDRI |
| [SPSS](https://www.ibm.com/products/spss-statistics?utm_content=SRCWW&p1=Search&p4=43700050715561164&p5=e&gclid=EAIaIQobChMIt-eJ2_Wo-QIV2vvjBx1mQwE9EAAYASAAEgJ0vfD_BwE&gclsrc=aw.ds) |Software used to write scripts to conduct data analysis.| N/A | At the lab | ONDRI, EpLink |
| [Metaboanalyst.ca](https://www.metaboanalyst.ca/) | Website used for data analysis of blood sample data. <details><summary>License</summary>GNU General Public License Version 2.0</details> <details><summary>Tool Citation(s)</summary>MetaboAnalyst (RRID:SCR_015539)</details> <details><summary>Relevant Publications</summary>[Chong et al., 2019](https://doi.org/10.1002/cpbi.86), [Pang et al., 2020](https://doi.org/10.3390/metabo10050186), [Pang, Chong, et al., 2021](https://doi.org/10.1093/nar/gkab382), [Pang, Zhou, et al., 2021](https://doi.org/10.3390/metabo11010044), [Pang et al., 2022](https://doi.org/10.1038/s41596-022-00710-w)</details> | N/A | Web-based | ONDRI |
| [impute lcmd (R package)](https://rdrr.io/cran/imputeLCMD/) | R package used for data analysis (molecular missing data below LOD) <details><summary>License</summary>GNU General Public License Version 2.0 </details>  <details><summary>Tool Citation(s)</summary> Cosmin Lazar and Thomas Burger (2022). imputeLCMD: A Collection of Methods for Left-Censored Missing Data Imputation. R package version 2.1. https://CRAN.R-project.org/package=imputeLCMD </details> <details><summary>Relevant Publications</summary>[Srivastava et al., 2019](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7059686/), [Alfano et al., 2020](https://doi.org/10.1016/j.metabol.2020.154292),  [Palstrøm et al., 2020](https://doi.org/10.1007/978-1-4939-9744-2_13), [Shen et al., 2021](https://doi.org/10.21203/rs.3.rs-298864/v1)</details> | [pcaMethods](https://www.bioconductor.org/packages/release/bioc/html/pcaMethods.html) and [impute](https://www.bioconductor.org/packages/release/bioc/html/impute.html) R packages | At the lab | ONDRI |
| [Mplus](http://www.statmodel.com/) | Software used for structural modeling of blood sample data. <details><summary>License</summary>Single-User License</details> <details><summary>Tool Citation</summary>Muthén, L.K. and Muthén, B.O. (1998-2017). Mplus User’s Guide. Eighth Edition. Los Angeles, CA: Muthén & Muthén</details><details><summary>Relevant Publications</summary> [Cella et al., 2007](https://doi.org/10.1097/01.mlr.0000258615.42478.55), [Babbott et al., 2014](https://doi.org/10.1136/amiajnl-2013-001875)</details> | N/A | At the lab | ONDRI |
    
</details>
