# Ontario Brain Institute Molecular Tools

Include a short introduction to the different types of molecular data OBI/Brain-CODE provides and maybe the researchers that collect the data?

<details><summary>Data Collection Tools</summary>
&nbsp 
    
| Tool/Pipeline | Description | Requirements | Compute Location | Research Program(s) |
| ---------------- | ----------- | --------------------------- | ----------- | ---------|
| [LabKey](https://www.labkey.com/) | Software used for life science data management. Data is collected in data form. <details><summary>License</summary>Apache Version 2.0</details> <details><summary>Tool Citation(s)</summary>**General Use**: Nelson EK, Piehler B, Eckels J, Rauch A, Bellew M, Hussey P, Ramsay S, Nathe C, Lum K, Krouse K, Stearns D, Connolly B, Skillman T, Igra M. LabKey Server: An open source platform for scientific data integration, analysis and collaboration. BMC Bioinformatics 2011 Mar 9; 12(1): 71. <br></br> **Proteomics**: Rauch A, Bellew M, Eng J, Fitzgibbon M, Holzman T, Hussey P, Igra M, Maclean B, Lin CW, Detter A, Fang R, Faca V, Gafken P, Zhang H, Whitaker J, States D, Hanash S, Paulovich A, McIntosh MW: Computational Proteomics Analysis System (CPAS):  An Extensible, Open-Source Analytic System for Evaluating and Publishing Proteomic Data and High Throughput Biological Experiments. Journal of Proteome Research 2006, 5:112-121. <br></br> **Flow**: Shulman N, Bellew M, Snelling G, Carter D, Huang Y, Li H, Self SG, McElrath MJ, De Rosa SC: Development of an automated analysis system for data from flow cytometric intracellular cytokine staining assays from clinical vaccine trials. Cytometry 2008, 73A:847-856.</details>| N/A | [Brain-CODE](https://www.braincode.ca/) | CAN-BIND |
| [ClinVar](https://www.ncbi.nlm.nih.gov/clinvar/) | Scientifically supported public archive of reports of the relationships among human variations and phenotypes <details><summary>License</summary>Permissive License</details> <details><summary>Tool Citation(s)</summary>Landrum MJ, Lee JM, Benson M, Brown GR, Chao C, Chitipiralla S, Gu B, Hart J, Hoffman D, Jang W, Karapetyan K, Katz K, Liu C, Maddipatla Z, Malheiro A, McDaniel K, Ovetsky M, Riley G, Zhou G, Holmes JB, Kattman BL, Maglott DR. ClinVar: improving access to variant interpretations and supporting evidence. Nucleic Acids Res . 2018 Jan 4. PubMed PMID: 29165669 </details> <details><summary>Relevant Publications</summary>To be filled</details> | N/A | Web-based | EpLink |
| [OMIM](https://www.omim.org/)| A comprehensive manual of human genes and genetic phenotypes <details><summary>Tool Citation(s)</summary>Online Mendelian Inheritance in Man, OMIM®. McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University (Baltimore, MD), {date}. World Wide Web URL: https://omim.org/ <br></br> Details on how to cite specific articles from OMIM can be found [here](https://www.omim.org/help/faq#1_8:~:text=Citing%20a%20specific%20entry%20in%20OMIM%3A)</details> <details><summary>Relevant Publications</summary></details>| N/A |  Web-based | EpLink |
| [Varsome](https://varsome.com/) | A variant knowledge community, data aggregator, and variant data discovery tool <details><summary>Tool Citation(s)</summary>VarSome: the human genomic variant search engine, Christos Kopanos, Vasilis Tsiolkas, Alexandros Kouris, Charles E Chapple, Monica Albarca Aguilera, Richard Meyer, Andreas Massouras, Bioinformatics, Volume 35, Issue 11, 1 June 2019, Pages 1978–1980, https://doi.org/10.1093/bioinformatics/bty897 </details>| N/A |  Web-based | EpLink |
 
</details>

<details><summary>Data Curation and Processing Tools</summary>
&nbsp
    
| Tool/Pipeline | Description | Requirements | Compute Location | Research Program(s) |
| ---------------- | ----------- | --------------------------- | ----------- | ---------|
| [Plink](https://github.com/GELOG/plink) | genome association analysis toolset that can be used to conduct a range of large scale analyses <details><summary>License</summary>GNU General Public License Version 2.0</details>| N/A | At the lab | ONDRI |
| Meds Tool | Excel based tool that performs drug categorization.| N/A | At the lab | ONDRI |
| [Simoa](https://www.svarlifescience.com/knowledge/technologies/simoa-bead-technology#:~:text=Simoa%20is%20an%20ultra%2Dsensitive,in%20most%20common%20sample%20types.) | Tool used for molecular classification of amyloid and Tau proteins | N/A | [Brain-CODE](https://www.braincode.ca/) | ONDRI |
| [Masshunter](https://www.agilent.com/en/product/software-informatics/mass-spectrometry-software?gclid=EAIaIQobChMI0drnxpWZ-QIV_T6tBh20DQ7pEAAYASAAEgJ7MfD_BwE&gclsrc=aw.ds) | Quantitation tool upstream of data upload <details><summary>License</summary>Customary non-commercial license <br> https://www.agilent.com/cs/library/quickreference/public/Agilent_Software_EULA.pdf</details>| N/A | At the lab | ONDRI |
| [Adobe Acrobat Pro 2020](https://www.adobe.com/ca/acrobat/complete-pdf-solution.html?mv=search&ef_id=EAIaIQobChMIz-2g35WZ-QIVIz2tBh1spgDtEAAYASAAEgLHzPD_BwE:G:s&s_kwcid=AL!3085!3!495116485612!e!!g!!adobe%20acrobat%20pro!12198071516!116786778653&gclid=EAIaIQobChMIz-2g35WZ-QIVIz2tBh1spgDtEAAYASAAEgLHzPD_BwE) | Software used to redact all personal health information (PHI) from genetic reports| N/A | At the lab | EpLink |
</details>

<details><summary>Data Analysis Tools</summary>
&nbsp 
    
| Tool/Pipeline | Description | Requirements | Compute Location | Research Program(s) |
| ---------------- | ----------- | --------------------------- | ----------- | ---------|
| RStudio |Software used to write scripts to conduct data analysis.| N/A | At the lab | ONDRI |
| SPSS |Software used to write scripts to conduct data analysis.| N/A | At the lab | ONDRI, EpLink |
| [Metaboanalyst.ca](https://www.metaboanalyst.ca/) | Website used for data analysis of blood sample data. <details><summary>License</summary>GNU General Public License Version 2.0</details> <details><summary>Tool Citation</summary></details> <details><summary>Relevant Publications</summary>take first 5 from https://www.metaboanalyst.ca/MetaboAnalyst/docs/Publications.xhtml </details> | N/A | Web-based | ONDRI |
| [impute lcmd (R package)](https://rdrr.io/cran/imputeLCMD/) | R package used for data analysis (molecular missing data below LOD) <details><summary>License</summary>GNU General Public License Version 2.0 </details>  <details><summary>Tool Citation</summary> Cosmin Lazar and Thomas Burger (2022). imputeLCMD: A Collection of Methods for Left-Censored Missing Data Imputation. R package version 2.1. https://CRAN.R-project.org/package=imputeLCMD | [pcaMethods](https://www.bioconductor.org/packages/release/bioc/html/pcaMethods.html) and [impute](https://www.bioconductor.org/packages/release/bioc/html/impute.html) R packages | At the lab | ONDRI |
| [Mplus](http://www.statmodel.com/) | Software used for structural modeling of blood sample data. <details><summary>License</summary>Single-User License</details> <details><summary>Tool Citation</summary>Muthén, L.K. and Muthén, B.O. (1998-2017). Mplus User’s Guide. Eighth Edition. Los Angeles, CA: Muthén & Muthén</details><details><summary>Relevant Publications</summary>To be filled</details> | N/A | At the lab | ONDRI |
   
</details>
